Fusion 4.0.2

w32 opened the pull request:
Fusion Server version changed to 4.0.2 and ubuntu image to 16.04 instead of latest(currently 18.04 - not works on it)

piccoloa committed:
first commit to branch

ozskywalker committed:
Merge pull request #9 from piccoloa/4.0.1

Upgrade to Fusion 4.0.1

w32 committed:
Changed fusion server version to 4.0.2, changed fusion sources URL

w32 committed:
Changed Fuison Server version into actual in README.txt

ozskywalker commented:
Thanks @w32 - let me test it out this side and merge it in!

ozskywalker commented:
Changes look good, and this works for single node ... but not multi-node, because of changes in zookeeper 3.5.3.   Will complete the work and perform a final change to this PR before merging in.

ozskywalker committed:
Updated build/deploy scripts & zookeeper tweaks

* Updated ports
* Separated out zookeeper cfg changes from build within Dockerfile
* reconfigEnable set to true (needed for dynamic reconfig with zookeeper 3.5.3+)
* Added additional waits via wait-for-it into masternode script for zookeeper instances on slave nodes
* Zookeeper console threshold increased from INFO to WARN (using change-zookeeper-log4j.sh)
* Updates to README.md, incl. software version numbers

Unfouranetly with the latest zookeeper build - multi-node doesn't work for now... will just have to live with it.

ozskywalker commented:
Docker Hub image has been updated (and the previous version tagged correctly as 4.0.1)

https://hub.docker.com/r/failathon/docker-fusion/tags/
